docs/source/developer-docs/contrib_guide_code.rst

```diff
@@ -55,20 +62,26 @@ be able to lend you assistance.
    reside and leave a comment in the file describing what issue you are trying
    to address.
 
-5. Open a pull request to the repository with [WIP] appended to the front
+5. Open a pull request to the repository with ``[WIP]`` appended to the front
    so that the core team is aware that you are actively pursuing the issue.
    When creating a pull request, make sure that the title clearly and concisely
    described what your code does. For example, we might use the title "Updated
    error message on ExampleException". In the body of the pull request, make sure
    that you include the phrase "Closes #issue-number-here", where the issue number is
    the issue number of the issue that you are addressing in this PR.
 
+   Feel free to open the PR as early as possible, you can get sometime early
```

Comment:
Feel free to open a PR as early as possible. Getting early feedback on your approach will save you time and prevent the need for an extensive refactor later.